server/s3_motr_rw_common.cc

```diff
@@ -195,6 +195,53 @@ void s3_motr_op_stable(struct m0_op *op) {
   s3_log(S3_LOG_DEBUG, request_id, "%s Exit", __func__);
 }
 
+// Motr callbacks, run in motr thread
+void s3_motr_op_executed(struct m0_op *op) {
```

Comment:
Function defined but never used.  I think, this callback should be used depending on callback type passed in _impl() method in this PR.

Reply:
@pranavPawar07 : How our handling in callback different for different callback type? In my opinion, there is no difference in executed and stable callback. So, why can't we pass 'stable' when callback type is 'executed'?

Reply:
@DPG17 for now, we are using the only stable callback. In the future, if we want to switch to another callback, e.g executed, we should be able to do that, that's what the purpose of this ticket is I think to just create a blueprint for future use. Actual implementation will be done in future sprints, I might be wrong here. @t7ko-seagate might help here.